client.go

```diff
@@ -436,12 +431,12 @@ func (c *Client) ContainerService() containers.Store {
 
 // ContentStore returns the underlying content Store
 func (c *Client) ContentStore() content.Store {
-	return contentservice.NewStoreFromClient(contentapi.NewContentClient(c.conn))
+	return NewContentStoreFromClient(contentapi.NewContentClient(c.conn))
 }
 
 // SnapshotService returns the underlying snapshotter for the provided snapshotter name
 func (c *Client) SnapshotService(snapshotterName string) snapshot.Snapshotter {
-	return snapshotservice.NewSnapshotterFromClient(snapshotapi.NewSnapshotsClient(c.conn), snapshotterName)
+	return NewSnapshotterFromClient(snapshotapi.NewSnapshotsClient(c.conn), snapshotterName)
```

Comment:
maybe `NewSnapshotterFromGRPCClient`? (so as to make sure we are not referring to `containerd.Client` here)